ha/setup/ha_setup.py

```diff
@@ -718,7 +718,12 @@ def remove_config_files(self):
         """
         Remove file created by ha.
         """
-        CleanupCmd.remove_file(const.CONFIG_DIR)
+        files = [const.CONFIG_DIR]
```

Comment:
This is a directory. It should be named accordingly, should be named like config_dir.

Reply:
Here in these list i stored both directory and pcsd files so i named it as list.

Reply:
Ok